exporter/loggingexporter/README.md

```diff
@@ -14,9 +14,9 @@ Supported pipeline types: traces, metrics, logs
 
 The following settings are optional:
 
-- `loglevel` (default = `info`): the log level of the logging export
-  (debug|info|warn|error). When set to `debug`, pipeline data is verbosely
-  logged.
+- `verbosity` (default = `normal`): the verbosity of the logging export
+  (detailed|normal|basic). When set to `detailed`, pipeline data is verbosely
+  logged. Anything above loglevel `info` is mapped to verbosity level `basic`.
```

Comment:
I would remove this last sentence, I don't think it's very useful for the end users.

```suggestion
  logged.
```